tutorials/learning-api.conf.yaml

```diff
@@ -1,31 +1,135 @@
-# This file relates tutorials in this folder to lesson URLs on the learning
-# platform (https://learning.quantum.ibm.com). See /tutorials/README.md for
-# instructions on adding a new tutorial.
-lessons:
-  - path: chsh-inequality
-    urlStaging: tutorials/1e8b063e-38c3-4525-824b-e4fbb1e675f9
-    urlProduction: tutorials/da7dbfe7-4ae7-4889-b78d-5dfd5263fb02
-  - path: grovers-algorithm
-    urlStaging: tutorials/b593867d-8aae-4cf7-b083-325028765fbd
-    urlProduction: tutorials/76db794e-a4a7-466e-bd2b-d5d5950a9279
-  - path: quantum-approximate-optimization-algorithm
-    urlStaging: tutorials/410c2e0d-004f-427d-a145-2014bb31899e
-    urlProduction: tutorials/8c5868ac-d82f-4ca4-ba9d-70d3d6344e80
-  - path: submitting-transpiled-circuits
-    urlStaging: tutorials/7dd1d5f9-3b6c-4bd5-9fb9-faeb728b44b2
-    urlProduction: tutorials/0f49821a-9de4-4932-a77a-ec3faf23cf55
-  - path: variational-quantum-eigensolver
-    urlStaging: tutorials/ef3b922f-0f32-42a3-8e3b-dc0fc18fd34c
-    urlProduction: tutorials/5e01f54c-0ef4-4003-8793-d6e5717110bb
-  - path: repeat-until-success
-    urlStaging: tutorials/caacbe11-a00f-4bbc-bc8b-953cf1d75b46
-    urlProduction: tutorials/5d88dda9-d990-4118-9950-8216528e8c8b
-  - path: build-repetition-codes
-    urlStaging: tutorials/46fc6326-bc12-4106-b334-7d693be34dd8
-    urlProduction: tutorials/7bd832af-5bc3-4382-b837-accb709d5e5c
-  - path: explore-composer
-    urlStaging: tutorials/28f53e87-52ba-4fa7-a1f9-9ad7f548a7e0
-    urlProduction: tutorials/967371a7-159d-459a-a7a6-210c55b14766
-  - path: combine-error-mitigation-options-with-the-estimator-primitive
-    urlStaging: tutorials/58d982db-9510-4815-a13c-c81abae1507f
-    urlProduction: tutorials/5f0507b3-3eae-4ccd-9a7a-74edd5a157a3
+# This file contains all the metadata for tutorials controlled by this repo.
+#
+# WARNING: Do not change any "slugs" until appropriate redirects are set up.
+# Speak to Abby Mitchell, Eric Arellano, or Frank Harkins to set up redirects.
+# See /tutorials/README.md for instructions on adding a new tutorial.
+#
+# Description of values:
+#
+#   title: The title of the page (top-level headings in the notebook are ignored)
+#   short_description: The description users will see in page previews (max 160 characters)
+#   slug: The last part of the URL; do not change this (see note above)
+#   local_path: The path to the tutorial folder, relative to `tutorials/`
+#   status: Can be "draft", "published", or "archived". Do not set to
+#           "archived" until appropriate redirects are set up.
+#   category: Can be "Workflow example" or "How-to"
+#   topics: List of topic names. Choose from
```

Comment:
Maybe add a note in this file that when you make changes to these two, update the setup test script.